lib/session/session.go

```diff
@@ -106,6 +106,8 @@ type Session struct {
 	AppName string `json:"app_name"`
 	// Owner is the name of the session owner, ie the one who created the session.
 	Owner string `json:"owner"`
+	// IsModeratedSession is true if the session requires moderation.
+	IsModeratedSession bool `json:"is_moderated_session"`
```

Comment:
Consider calling this `Moderated` instead?

`if session.Moderated` reads better IMO than `if session.IsModeratedSession`.

Reply:
I can get behind that. and that removes any "camelcase vs snakecase" argument too, why not